Fix Athena sample DAG

Alagappan opened the pull request:
### Summary of Changes

Made couple of changes to Athena sample DAG. 
 - PythonOperators in the DAG were not initialized properly. Fixed the DAG creation and set the dag structure explicitly
 - Renamed the Task Names to be consisted with other sample dags
 - Removed unused kwargs in method call. 

### CheckList
Make sure you have checked **all** steps below to ensure a timely review.
- [X] PR title addresses the issue accurately and concisely. Example: "Updates the version of Flask to v1.0.2"
    - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
- [X] PR includes a summary of changes. 
- [X] PR adds unit tests, updates existing unit tests, __OR__ documents why no test additions or modifications are needed.
- [X] In case of new functionality, my PR adds documentation that describes how to use it.
    - All the public functions and the classes in the PR contain docstrings that explain what it does
- [X] PR passes `make test`

Alagappan commented:
cc @feng-tao another one. athena dag didn't seem to be set up correctly. Please take a look and let me know

ravioactive commented:
Hi @Alagappan 
I have been trying to adapt this connector for my AWS environment. When I used this as is, I get a `botocore.exceptions.ClientError: An error occurred (UnrecognizedClientException) when calling the StartQueryExecution operation: The security token included in the request is invalid.`
I looked into this exception a little bit and there were suggestions to pass in a session token to the connection url, and I have been trying different methods, but I have had no success.
Can you suggest how to include the session token into the jdbc athena url?

Alagappan commented:
Hello @ravioactive 
I haven't worked with Athena. I just updated this file to use standard naming conventions and modified the DAG structure a bit. Could you please check your issue on Amundsen slack channel to see if someone can help? Thanks!
